articles/user-profile/user-impersonation.md

```diff
@@ -2,57 +2,58 @@
 - Unlimited customization on who can impersonate who, when, depending on whatever context, using our [Rules](/rules) engine. In a Rule, you have access to `user.impersonated` (the impersonated login) and `user.impersonator` (the impersonating login) and you can write arbitrary Javascript to define how it works.
 
 ::: note
-  Any Rules that you've implemented will run when you impersonate a user, including any actions that update the user.
+Any Rules that you've implemented will run when you impersonate a user, including any actions that update the user.
 :::
 
 ## Use the Dashboard
 
-Navigate to the [Users](${manage_url}/#/users) page in the Management Dashboard and select the user you want to login as. Click on the __Sign in as User__ and select the client you want to log into using the dropdown menu.
+Navigate to the [Users](${manage_url}/#/users) page in the Management Dashboard and select the user you want to login as. Click on the "Sign in as User: and select the client you want to log into using the dropdown menu.
 
-![Click Sign in as User](/media/articles/user-profile/signin-as-user-01.png)
+![Click Sign in as User](/media/articles/user-profile/user2.png)
 
 ::: panel I can't see this button
 In order to see this button the following conditions should apply:
+
 - Impersonation should be enabled for your account (see panel at the top of this page)
-- The Clients registered in the account must have at least one __callback URL__ listed
+- The Clients registered in the account must have at least one callback URL listed
 - The Clients must have the connections turned on that the users who are to be impersonated belong to
 :::
 
 A popup displays the URL to be used in order to impersonate the user. You can choose either to copy the URL into the clipboard (white button) or open it in a separate browser tab/window (blue button).
 
-![Links for User Impersonation](/media/articles/user-profile/signin-as-user-02.png)
+![Links for User Impersonation](/media/articles/user-profile/user3.png)
 
-::: note
-Impersonating a user using the Dashboard will not return a [ID Token](/tokens/id-token) to your application by default. To achieve this, call the [impersonation endpoint](/api/authentication/reference#impersonation) manually or in the [Advanced Settings](#advanced-settings). If you call the endpoint manually, add `additionalParameters.scope: "openid"` to the request body.
+::: panel Acquiring a Token
+Impersonating a user using the [Dashboard](${manage_url}) will not return an [ID Token](/tokens/id-token) to your application by default. There are two ways to achieve this. You can alter the "Response Type" setting in the impersonation menu's [Advanced Settings](#advanced-settings) from "Code" to "Token" (Sign in as user -> Show Advanced Settings). Alternatively, you can add `additionalParameters.scope: "openid"` to the request body while calling the [impersonation endpoint](/api/authentication/reference#impersonation) manually.
```

Comment:
Use bold for field names, code formatting for values, ie: You can alter the __Response Type__ setting in the impersonation menu's [Advanced Settings](#advanced-settings) from `Code` to `Token` (__Sign in as user__ > __Show Advanced Settings__)